### PR TITLE
Mathbox

### PR DESCRIPTION
**Pull request by benjub**

### CODE REVIEW

**Comment by wlammen** [comment]
You usually add a bj- prefix to theorem names in your mathbox.  This time some are not according to this rule.  Is this intentional?

**Comment by benjub** [comment]
> You usually add a bj- prefix to theorem names in your mathbox. This time some are not according to this rule. Is this intentional?

Thanks, I'm fixing it.

File: set.mm [hunk 1]
```diff
@@ -527093,6 +527118,180 @@ element of the complex projective line (Riemann sphere) to its inverse.
...
+      bj-fununsn2 $p |- ( ph -> ( F ` B ) = C ) $=
+        ( cfv cop csn cun uncom syl6eq bj-funun wcel wceq fvsng syl2anc eqtrd )
+        ABDLBBCMNZLZCABDUDEADEUDOUDEOHEUDPQIRABFSCGSUECTJKBCFGUAUBUC $.
+    $}
+  $}
+
+  $( An element of a difference with a singleton is not equal to the element of
+     that singleton.  Note that ` ( -. A e. { C } -> -. A = C ) ` need not hold
+     if ` A ` is a proper class.  (Contributed by BJ, 18-Mar-2023.) $)
+  eldifsnneq $p |- ( A e. ( B \ { C } ) -> -. A = C ) $=
```
Comment (wlammen) [inline set.mm:527218]: Still a theorem not following your naming style

Comment (benjub) [reply set.mm:527218]: Oh, yes. I plan to move it to Main soon anyway, so it can stay like that.

Comment (avekens) [reply set.mm:527218]: This is, in principle, the same as ~eldifsni. I think we do not need both in main.

**Review by david-a-wheeler — APPROVED** [review]

**Comment by david-a-wheeler** [comment]
+1. As long as they stay in *mathboxes* and the changes still pass all checks, I'm generally fine with additions.

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:527218]: Oh, I see, the MM-PA didn't catch it when I tried to prove the other theorem, so I added it without looking further.

**Pull request merged**